<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>452</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>452</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>452</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Port Forward Tool</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 1px;
border-style: solid;
border-color: black;</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0px;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Add</string>
     </attribute>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>76</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_AddressLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_PortLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="nftablesRadioButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>nftables</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="iptablesRadioButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>iptables</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Remove</string>
     </attribute>
     <widget class="QPushButton" name="removeButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>76</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;</string>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Listenadderss</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Listenport</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="del_listenAddressLineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="del_listenPortLineEdit_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 1px;
border-style: solid;
border-color: black;</string>
   </property>
   <widget class="QTextEdit" name="ConsoleOutput">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0px;</string>
    </property>
    <property name="text">
     <string>Console</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshConsoleButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>‚ü≥</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
